addons/l10n_ca/data/account_tax_data.xml

```diff
@@ -84,7 +84,6 @@
         <field name="amount">100</field>
         <field name="amount_type">group</field>
         <field name="children_tax_ids" eval="[(6,0,[ref('pst_bc_sale_en'), ref('gstpst_sale_bc_gst_en')])]"/>
```

Comment:
@var-odoo Are there still children_tax_ids in 12.3?

Reply:
@jco-odoo yes, in cases when regular taxes(sale/purchase type taxes) has to be used as children taxes, at that time we do not create separate tax repartition lines and instead we directly refer regular taxes as children taxes and make their parent tax a `group` tax. See such one case <a href="https://github.com/odoo/odoo/blob/master/addons/l10n_co/data/account_tax_template.xml#L1616">here</a>